/**:
  ros__parameters:
    # Exploration Area (Related points)
    x_min: 0.0
    x_max: 95.0
    y_min: -9.0
    y_max: 48.0
    z_min: 0.3
    z_max: 1.5
    
    # Tree Detection Params
    treeClusterTolerance: 0.3
    treeClusterMinSize: 5
    treeClusterMaxSize: 10000
    
    # Row Detection Params
    rowClusterTolerance: 4.0
    rowClusterMinSize: 3
    rowClusterMaxSize: 70
    cosThetaThreshold: 0.9
    dot2LineThreshold: 2.0